src/main/resources/kibana-rest.yml

```
@@ -42,6 +42,10 @@ kibana_stats:
   versions:
     ">= 6.5.0": "/api/stats"
 
+kibana_status:
+  versions:
+    ">= 4.2.0": "/api/status"
```

Most of the rest of the diag will not run without a lot of the 5.0+ changes in X-Pack, so I don't want this to imply that it will.

```suggestion
    ">= 5.0.0": "/api/status"
```